chains/orchestrator-paras/node/src/command.rs

```diff
@@ -167,13 +167,26 @@ impl SubstrateCli for RelayChainCli {
     }
 
     fn load_spec(&self, id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
+        const STARLIGHT_RAW_SPECS: &[u8] =
+            include_bytes!("../../../../specs/solochain/starlight-raw-specs.json");
+
         match id {
             "westend_moonbase_relay_testnet" => Ok(Box::new(WestendChainSpec::from_json_bytes(
                 &include_bytes!("../../../../specs/dancebox/alphanet-relay-raw-specs.json")[..],
             )?)),
-            // If we are not using a moonbeam-centric pre-baked relay spec, then fall back to the
+            // Default to starlight if this is a solochain node. Else default to polkadot default.
+            "" if self.solochain => Ok(Box::new(GenericChainSpec::from_json_bytes(
+                &STARLIGHT_RAW_SPECS[..],
+            )?)),
+            "starlight" => Ok(Box::new(GenericChainSpec::from_json_bytes(
```

Comment:
Here do we want starlight or tanssi? I can add both as an alias @girazoki

Reply:
tanssi & starlight yes